web_src/js/features/install.ts

```diff
@@ -1,6 +1,8 @@
 import {hideElem, showElem} from '../utils/dom.ts';
 import {GET} from '../modules/fetch.ts';
 
+type DatabaseType = 'mysql' | 'postgres' | 'mssql' | 'sqlite3';
```

Comment:
Should we define the types here? Once it is changed by the backend, we may forget we also need to change it.

Reply:
Don't need to define so specific, I will change these to `string`.

Reply:
On the other hand, if you change these db types, the file will need to be touched in any case likely to update `defaultDbHosts`.

Reply:
I changed it to `string`: https://github.com/go-gitea/gitea/pull/33322/commits/c3aea25084a7da75a878d3177062b2b982fb411a.